org.eclipse.scout.rt.client/src/main/java/org/eclipse/scout/rt/client/ui/form/AbstractForm.java

```diff
@@ -2365,6 +2381,31 @@ public void loadFromXml(Element root) {
         }
       }
     }, ITabBox.class);
+    return success;
+  }
+
+  protected boolean isXmlFieldWithStoredValue(Element xField) {
+    // heuristic for inner forms: do not check recursively for field values. simply assume that an inner form has a value
+    return isXmlWrappedFormField(xField) ||
```

Comment:
I am not sure if this is actually complete. Especially considering one can write their own special fields.